pages/nft-book-store/new.vue

```diff
@@ -252,23 +304,25 @@ function sanitizeHtml (html: string) {
 
 async function onSubmit () {
```

Comment:
How about separating `onSubmit()` into 2 sub-functions for new and edit
e.g. `submitNewClass()`, `submitEditedClass()`

```suggestion
async function onSubmit () {
  return this.isEditMode.value ? this.submitEditedClass() : this.submitNewClass();
}
```